src/hooks/useAuth.js

```diff
@@ -40,7 +40,7 @@ export function useAuth() {
     }
   }, []);
 
-  const withdraw = useCallback(() => {
+  const withdraw = useCallback((user) => {
     Auth.withdraw(user);
   }, []);
```

Comment:
로그인 이슈가 고쳐진 이후이기 때문에 ProfilePage에 접근한다는건 useAuth훅 내에 user가 반드시 존재한다는 것이니까 user를 훅 외부에서 받지말고 내부에 있는 user로 처리가 가능할듯

```suggestion
  const withdraw = useCallback(() => {
    Auth.withdraw(user);
  }, [user]);
```